packages/core/core/src/config.ts

```diff
@@ -183,7 +186,9 @@ export class Config {
       config: this,
       context: this.context,
       env: this.env,
-      version: this.version
+      version: this.version,
+      client: this.client,
+      locale: this.locale
```

Comment:
If `client` and `locale` are part of `config`, there is no need to add it to Plugins and/or Plugin, since plugins have access to the config.